addr_two_pkey.key

```diff
@@ -0,0 +1 @@
+7b2254797065223a22736563703235366b31222c22507269766174654b6579223a223546545633762f322f62527030753231544b466b6f386d2f566d5632434e6359334b444b303848623456633d227d
```

Comment:
Maybe add `test` to the file name to make it clear it's not sth important being commited by mistake

Reply:
added